nuget/bootstrap.sass.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
   <metadata>
     <id>bootstrap.sass</id>
-    <version>4.0.0-alpha.3</version>
```

Comment:
less churn, the nuget publishing script sets the version to match the project.json file so now there is one less thing to update for version bumps.